### PR TITLE
PR to update the validation changes in pierapi

### PR DESCRIPTION
**Pull request by mjpierfinance**
Makes the changes to docs that are present in https://github.com/pierdeveloper/pierapi/pull/170

### CODE REVIEW

**Review by hegeh0g — APPROVED** [review]

File: fern/definition/api.yml [hunk 1]
```diff
@@ -1,5 +1,6 @@
 name: api
 display-name: Pier API
+# TODO: add non-basic auth and the token generation endpoint in docs
```
Comment (mjpierfinance) [inline fern/definition/api.yml:3]: @lesjonesiv @hegeh0g need to discuss based on post in `#eng` before adding

File: fern/docs.yml [hunk 2]
```diff
@@ -23,13 +23,13 @@ navigation:
...
-  accentPrimary: 
+  accentPrimary:
     dark: '#aa8ff6'
     light: '#6f5ea0'
-logo: 
+logo:
   dark: ./docs/assets/logo-white.png
   light: ./docs/assets/logo-black.png
-  height: 35
+  height: 25
```
Comment (mjpierfinance) [inline fern/docs.yml:32]: Based on Deep's suggestion

File: fern/docs/pages/quickstart.mdx [hunk 3]
```diff
@@ -13,7 +13,7 @@ The sandbox is designed to mimic the production environment, with a few exceptio
 
 ### Postman Collection
 
-You can fork our Postman collection to immediately start interacting with the API. You'll need to set a few variables to get started:
+You can fork our [Postman Collection](https://www.postman.com/martian-comet-959825/workspace/pier-public/collection/14754440-cf96aaa5-d588-4a09-b4a0-66c32b77e33c?action=share&creator=14754440) to immediately start interacting with the API. You'll need to set a few variables to get started:
```
Comment (mjpierfinance) [inline fern/docs/pages/quickstart.mdx:16]: Add postman collection

File: fern/docs/pages/quickstart.mdx [hunk 4]
```diff
@@ -23,7 +23,7 @@ You can fork our Postman collection to immediately start interacting with the AP
 
 ### Authentication
 
-Pier uses basic auth to authenticate API requests. All calls should be made with the `username` field set to your `client_id` and the `password` field set to your `api_secret`.
+Pier uses basic auth to authenticate API requests. All calls should be made with the `username` field set to your `client_id` and the `password` field set to your `api_secret`. Make sure to `base64` encode the `username:password` portion.
```
Comment (mjpierfinance) [inline fern/docs/pages/quickstart.mdx:26]: Felt mention of b64 encode is helpful

File: fern/definition/applications.yml [hunk 5]
```diff
@@ -273,53 +305,31 @@ types:
         docs: The annual fee charged for the credit line (e.g. '5000' for $50.00)
       billing_cycle:
         type: optional<integer>
-        docs: |
+        docs: >
```
Comment (lesjonesiv) [inline fern/definition/applications.yml:308]: What does this symbol do?

**Review by lesjonesiv — COMMENTED** [review]
Woof. Thanks for doing this.

File: fern/definition/applications.yml (hunk 5 above)
Comment (mjpierfinance) [reply fern/definition/applications.yml:308]: wraps the docs instead of relying on the newline char

**Pull request merged**